src/nupic/regions/RecordSensor.py

```diff
@@ -556,3 +572,4 @@ def readFromProto(cls, proto):
     instance.numCategories = proto.numCategories
 
     return instance
+  
```

Comment:
make sure files end with a newline. Many editors handle this automatically but YMMV